roles/katello_devel/meta/main.yml

```diff
@@ -6,15 +6,19 @@ dependencies:
     foreman_server_repositories_katello: true
     foreman_server_repositories_foreman_client: true
   - role: ruby_scl
+    when: ansible_distribution_major_version == "7"
   - role: nodejs_scl
+    when: ansible_distribution_major_version == "7"
   - role: postgresql_scl
+    when: ansible_distribution_major_version == "7"
```

Comment:
should this instead become a `devel_scls` meta-role? (not blocking this PR)

Reply:
Maybe? I do not see the need for that additional layer of indirection presently.